pkg/metrics/aws/instruments.go

```diff
@@ -13,6 +13,11 @@ const (
 
 	metricAPIRequestsTotal          = "api_requests_total"
 	metricAPIRequestDurationSeconds = "api_request_duration_seconds"
+
+	metricAPIAuthErrorsTotal      = "api_call_auth_errors_total"
+	metricAPILimitExceededTotal   = "api_call_limit_exceeded_total"
+	metricAPIThrottledTotal       = "api_call_throttled_total"
+	metricAPIValidationErrorTotal = "api_call_validation_error_total"
```

Comment:
nit: maybe to be consistent on using either `Error` or `Errors` for the metrics naming.